apps/antalmanac/src/lib/api/endpoints.ts

```diff
@@ -1,4 +1,10 @@
 function endpointTransform(path: string) {
+    if (import.meta.env.VITE_ENDPOINT) {
```

Comment:
Public environment variables with a certain prefix (`VITE` by default) are exposed by [vite](https://vitejs.dev/guide/env-and-mode.html)

But yeah, it would be a good idea to include any environment variables we use in the `.env` file to give a full picture of what we're working with.